presto-native-execution/presto_cpp/main/common/Configs.cpp

```diff
@@ -142,21 +142,36 @@ ConfigBase::ConfigBase()
     : config_(std::make_unique<velox::core::MemConfig>()) {}
 
 void ConfigBase::initialize(const std::string& filePath) {
-  // See if we want to create a mutable config.
-  auto values = util::readConfig(fs::path(filePath));
+  std::unordered_map<std::string, std::string> configFromFile =
```

Comment:
I think the guidelines of Velox is to use 'auto'.
Also, as we operate with config classes and this is a map, maybe call it 'propMapFromFile'?